test/integration/submission/submission.spec.ts

```diff
@@ -844,6 +848,151 @@ describe('Submission Api', () => {
     });
   });
 
+  describe('clinical-submission: clear', function() {
+    const programId = 'ABCD-EF';
+    let donor: any;
+    let submissionVersion: string;
+
+    const uploadSubmission = async () => {
+      let file: Buffer;
+      try {
+        file = fs.readFileSync(__dirname + '/donor.tsv');
+      } catch (err) {
+        return err;
+      }
+
+      await chai
+        .request(app)
+        .post(`/submission/program/${programId}/clinical/upload`)
+        .auth(JWT_CLINICALSVCADMIN, { type: 'bearer' })
+        .attach('clinicalFiles', file, 'donor.tsv')
+        .then((res: any) => {
+          submissionVersion = res.body.submission.version;
+        })
+        .catch(err => chai.assert.fail(err));
+
+      try {
+        file = fs.readFileSync(__dirname + '/specimen.tsv');
+      } catch (err) {
+        return err;
+      }
+
+      await chai
```

Comment:
it's possible to attach multiple files to one request by multiple `.attach` so you don't have to do them separately, unless if there is a reason for it...